src/EnergyPlus/SingleDuct.cc

```diff
@@ -2747,7 +2747,7 @@ namespace SingleDuct {
 		UpdateSys( SysNum );
 
 		// At the current air mass flow rate, calculate heating coil load
-		QActualHeating = QToHeatSetPt - MassFlow * CpAirZn * ( SysInlet( SysNum ).AirTemp - ZoneTemp ); // reheat needed
+		QActualHeating = QToHeatSetPt - MassFlow * ( CpAirSysIn * SysInlet( SysNum ).AirTemp - CpAirZn * ZoneTemp ); // reheat needed
```

Comment:
Would it be better to change the way RemainingOutputRequired is calculated instead of changing all the other places where capacity is calculated?